Cargo.toml

```diff
@@ -46,7 +46,6 @@ lazy_static = "1.4.0"
 pathdiff = "^0.2"
 md5 = "^0.7"
 symlink = "^0.1.0"
-cssparser = "0.29.6"
```

Comment:
since regex matches work, it felt unnecessary to add this. I was stuck on this problem for a while due to lifetimes and FnOnce closures, so my implementation does not utilize cssparser...this PR is ideally more lightweight without this crate